src/styles/header.scss

```diff
@@ -0,0 +1,82 @@
+@import 'variables';
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: $white;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0 , 0.05);
+  flex-shrink: 0;
+}
+
+.logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.logo__image {
+  height: 100%;
+  width: 100%;
+}
+
+.nav__list {
+  display: flex;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+.nav__item {
+  margin-left: 20px;
+  transition: transform 300ms;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__link.is-active {
+  color: #00acdc;
```

Comment:
Hardcoded color is used. To maintain consistency, consider using a color variable instead.